coral-component-table/src/scripts/Table.js

```diff
@@ -2480,8 +2501,24 @@ class Table extends BaseComponent(HTMLTableElement) {
       wrapper.remove();
     }
   
+    
     // Append frag
+    let liveRegion = this.querySelector('table-sort-description');
```

Comment:
why do you use a custom name instead of div for example ?

Reply:
It could be just a `div` with a `className`, but this seemed how things were done for Coral components.